plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.h

```diff
@@ -1061,19 +1018,26 @@ class ProxyShape
   std::string generateTranslatorId(UsdPrim prim) override
    { return m_translatorManufacture.generateTranslatorId(prim); }
 
+public:
+  bool isLockPrimFeatureActive() const
+  {
+    bool ignoreLockPrims = MGlobal::optionVarIntValue("AL_usdmaya_ignoreLockPrims");
+    //The lock Prim functionality is a UI thing - no need to have it on in batch mode
+    //However, this also causes the tests to fail which is bad
```

Comment:
Why do the tests fail?  What's the interaction?

Reply:
When we run the tests on our build server, ````MGlobal::mayaState() == MGlobal::kInteractive``` will return false, which would end up disabling the meta-data processing entirely. Not overly useful when you've written tests to test said functionality.